### DISCUSSION TITLE
chapter_heap/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#538] [Announcements]
# chapter_heap/

一本动画图解、能运行、可提问的数据结构与算法入门书

https://www.hello-algo.com/chapter_heap/

**Comment by suiyuano** [comment]
堆这一章怎么没人

**Reply by krahets** [reply]
现在有我们俩了

**Reply by FOXnosmile** [reply]
代码已经拉到本地了，该怎么演示呀，想试一试动画演示

**Reply by krahets** [reply]
@FOXnosmile Hi，你是指[项目部署](https://www.hello-algo.com/chapter_appendix/contribution/#3-docker)吗，还是[运行代码](https://www.hello-algo.com/chapter_appendix/installation/)？

**Reply by meilaoliu** [reply]
@FOXnosmile 动画演示是用的一个网站  只能在网页 本地应该演示不了

**Comment by yuanshanchang** [comment]
作者你好，图片使用AI画的吗

**Comment by pinglanke12138** [comment]
还有8天期末，现在梭哈

**Reply by Jppp33** [reply]
是的

**Comment by acmu** [comment]
配图真的好看👍

**Comment by coderguai** [comment]
感谢！

**Comment by 490273789** [comment]
感谢

**Comment by brocyu** [comment]
能看懂，感谢

**Comment by xiaozhuge123** [comment]
day07

**Comment by it-ham** [comment]
太棒了
